pages/about.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="/styles/style.css" rel="stylesheet">
+ 
 </head>
 <body>
-  
-</body>
+<section class = "flex-container">
+<h1><b><i><u>Who is She?</u></i></b></h1>
+
+<h2>
+<p><em>Somone's Mom</em></p>
+</h2>
+<ul>
+<li>I have a 10 month old daughter named Zaela aka my<b><i>WORLD!</i></b></li> 
+<li>She will be walking any day now :)</li>  
+</ul>
+<h2><em>Somone's Partner</em></h2>
+<ul>
+<li>My partner's name is Isaac</li> 
+<li>Almost 4 years, although we moved in together during lockdown so it seems like longer</li>
+</ul>
+<h2><em>Somone's Future Developer</em></h2>
+<ul>
+<li>I come from a background of civil service - and want to use technology to give back to the community after Ada</li> 
+<li>Use python fundamentals, API knowledge <u>and</u> learn front end to prepare for fulltime work in 2023.</li>
+</ul>
+<footer>
+<nav>
+    Links to:
+<ul>
+<li>Portfolio: <a href = "http://127.0.0.1:5500/pages/portfolio.html"><em>here</em></a></li>
```

Comment:
When linking within a site, don't include the server part of the address `http://127.0.0.1:5500/`. Only include this when a link leaves your site. Because of the hard-coded 127.0.0.1, the links would be broken in a deployed site (since the address of the deployed site wouldn't be 127.0.0.1).

Reply:
You chose to omit listing the page itself in the links, which is absolutely valid, an davoidsoffering visitors a false choice. However, another common approach is to include all the pages in the nav section everywhere, but omit the link around the current page. By not changing the content showed in the nav links, it gives users a chance to build some spacial familiarity with the items in the navigation area, which can be helpful.